.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: cpp
+dist: trusty
+matrix:
+  include:
+  - os: linux
+    env: CONDA=3.6 RDKIT=2019.09.2
```

Comment:
I wouldn't go all the way to the patch release with the RDKit. That would require this to be updated with every RDKit patch. I would suggest just using `2019.09`

Reply:
ok, I added the patch version because I believe there are some final bug fixes you did for us that were only in that patch version.

Reply:
That's true, but by making it this explicit in the build matrix it would mean that you won't get the newest RDKit version when we release 2019.09.3 (and beyond)
I think the check you have in the code for the RDKit version is sufficient